### ISSUE TITLE
Past DCDW website links should include more context

### ISSUE THREAD

**Issue by joshkimux** [AIGAdc/dcdesignweek.org-11ty#22]
# Past DCDW website links should include more context

## Defect level
Feedback will follow a must, should, consider feedback framework:

- ❗️ **Must for if the feedback must be applied.** Failure to follow this feedback may either cause harm to users, block a disabled person from completing an important task, or result in legal risk (fails to meet WCAG 2.1 AA compliance).
- ⚠️ **Should if the feedback is best practice.** Following this feedback will significantly improve the usability for disabled people or reduce the complexity of accessibility barriers.
- ✔️ **Consider for suggestions or enhancements.** Think of these suggestions as reducing the hundreds of papercuts and microaggressive barriers disabled people may face when navigating the web. Going above and beyond here will make us a leader in accessibility. 

## Description
The 2016, 2017, 2018, and 2019 links lack purpose (why) and target (where). Screen reader users who prefer to navigate by the links list (essentially getting an isolated list of links) may be confused where `2016` on its own would go. They would need to effectively exit the links list view and navigate around the link to determine the context which is a bit of a chore.

<img width="988" alt="Screen Shot 2022-05-24 at 6 54 00 PM" src="https://user-images.githubusercontent.com/14154792/170145244-596e7a87-8e14-4c2e-88d9-32e923a3898f.png">

## Recommendation or success criteria
- [x] Use `aria-label="[insert year] DCDW Website"`
